### DISCUSSION TITLE
Invalid config for [sensor.wundergroundpws]: [api_key] is an invalid option for [sensor.wundergroundpws]

### DISCUSSION THREAD

**Discussion by claustro** [cytech/Home-Assistant-wundergroundpws#153] [Q&A] [answered]
I am breaking my head finding where I am wrong.
Tryend installing manually and trough HACS . Version 1.1.1.
my config in sensor:
  - platform: wundergroundpws
    api_key: xxxxxxxxxxxxxxxxxxxxxxxxx
    pws_id: xxxxxx
    numeric_precision: decimal
    monitored_conditions:
      - temp
      - obsTimeLocal
      - humidity
      - dewpt

Any idea?
Thank you
Andrea

**Comment by rudizl** [comment, marked as answer]
```
# Example configuration.yaml entry
# REQUIRED
wundergroundpws:
  api_key: YOUR_API_KEY
  pws_id: YOUR_STATION_ID

# Required to generate HASS weather Entity for weather forecast card
weather:
  - platform: wundergroundpws

# Required if you wish to add sensors
sensor:
  - platform: wundergroundpws
    monitored_conditions:
      - temp
      - dewpt
      - heatIndex

**Comment by claustro** [comment]
Thank you for taking time of answering me.
now I have in configuration .yaml

wundergroundpws:
  api_key: xxxxxxxxxxxxxxxxxxxxxxxxx
  pws_id: xxxxxx


in sensor.yaml
  - platform: wundergroundpws
    monitored_conditions:
      - temp
      - dewpt
      - heatIndex
      - 
I have no errors checking config and homeassitant starts regularly.
but I receive this error.

Unable to prepare setup for platform wundergroundpws.sensor: Unable to set up component.
10:13:17 – (ERRORE) setup.py
Error during setup of component wundergroundpws
10:13:17 – (ERRORE) wundergroundpws (integrazione personalizzata)
Check WUnderground API ('NO CURRENT RESULT',)
10:13:17 – (ERRORE) wundergroundpws (integrazione personalizzata)

I tryed to reinstal the integration trough HACS and even manually but the error persist. 
Any ide 

thank you
Andrea

**Comment by rudizl** [comment]
You've missed
```
weather:
  - platform: wundergroundpws
```
In configuration.yaml

**Comment by claustro** [comment]
Solved thank you very much !!

**Comment by rudizl** [comment]
Maybe it will be good idea that @cytech changes the default branch to `v.1.x.x`, because now the default is `master` with the old description of installation process.

**Reply by claustro** [reply]
you are right , I saw the readme inside the folder only after your suggestions

**Comment by cytech** [comment]
closed as outdated

**Comment by cytech** [comment]
closed as outdated
